cafebabel/templates/users/detail.html

```diff
@@ -8,7 +8,7 @@ <h1>{{ user.profile.name or '' }}</h1>
 
 <section>
   <h2>About</h2>
-  {{ profile_image(user) }}
+  {{ profile_image(user, dimensions='500x500') }}
```

Comment:
200x200

Reply:
You sure about that? Here we display the whole image profile, not the “icon” one.

Reply:
yep:
![capture d ecran 2018-04-11 a 15 49 41](https://user-images.githubusercontent.com/32518627/38620926-01280f3c-3da0-11e8-94fd-8222007b12af.png)

Reply:
Alright, in that case the default one is probably wrong to display small profile pictures

Reply:
default pic is also 200x200